src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -157,7 +157,18 @@ public function loadAnnotations(object $reflector): iterable
     {
         foreach ($reflector->getAttributes() as $attribute) {
             if ($this->isKnownAttribute($attribute->getName())) {
-                yield $attribute->newInstance();
+                try {
+                    yield $attribute->newInstance();
+                } catch (\Error $e) {
```

Comment:
I'm always mixed on catching `Error` but I guess there is a specific type you want to catch?

Reply:
should we throw when `get_class($e) !== Error::class`?

Reply:
I don't like to catch such wide error too. But this is the best type I guess:

![image](https://user-images.githubusercontent.com/408368/147657909-879f35c3-3d47-4632-9ba2-6f906846ce51.png)

Reply:
Can you ad the check to throw when `get_class($e) !== Error::class`?

Reply:
done (with $e::class, hope it's not an issue ; I forgot to re-read your comment before patchin)

For the ref: https://heap.space/xref/PHP-8.1/ext/reflection/php_reflection.c?r=59dd4fd7#6596